### PR TITLE
Fixed document create/edit form to not validate unless the user tries to save

### LINKED ISSUE #148: do not validate empty form when creating a new site
otherwise the mandatory fields will be marked red

### PR DESCRIPTION
**Pull request by invliD**
Fixes #148.

### CODE REVIEW

File: _1327/documents/utils.py [hunk 1]
```diff
@@ -41,26 +39,24 @@ def handle_edit(request, document):
...
 			autosaved = False
 
-		form_data = {
-			'title': document.title,
-			'text': text,
-		}
-		form = TextForm(form_data)
+		if 'restore' in request.GET and autosave is not None:
+			form_data = {
+				'text': autosave.text,
```
Comment (Nef10) [inline _1327/documents/utils.py]: you loose the old title of the page if you restore an autosave

Comment (invliD) [inline _1327/documents/utils.py]: It's now fixed in line 57. (The use of the ``initial`` dictionary overwrites the text, all other fields are still taken from the ``document`` model)

File: _1327/information_pages/models.py [hunk 2]
```diff
@@ -25,3 +26,10 @@ def get_edit_url(self):
 		return reverse('information_pages:edit', args=(self.url_title, ))
 
 reversion.register(InformationDocument, follow=["document_ptr"])
+
+
+class InformationDocumentForm(DocumentForm):
```
Comment (Bartzi) [inline _1327/information_pages/models.py]: is there a reason why this is in the `models.py`?

**Comment by Bartzi** [comment]
:+1:

**Pull request merged**